src/index.js

```diff
@@ -0,0 +1,116 @@
+import './style.css';
+
+const list = document.querySelector('.list');
+const inputTodo = document.querySelector('.add-todo');
+const addTodoBtn = document.querySelector('.submit');
+
+class TodoTask {
+  constructor(desp, arr, done = false) {
+    this.desp = desp;
+    this.index = arr.length;
+    this.done = done;
+  }
+}
+
+class TaskLisk {
+  constructor() {
+    this.tasks = [];
```

Comment:
- [ ] Again, from the requirements of this project, the task array should be rendered dynamically to the DOM on page load.